chcemvediet/apps/inforequests/admin.py

```diff
@@ -253,13 +253,51 @@ class InforequestEmailAdmin(admin.ModelAdmin):
             ]
     inlines = [
             ]
+    actions = [
+            u'delete_selected'
+    ]
 
     def get_queryset(self, request):
         queryset = super(InforequestEmailAdmin, self).get_queryset(request)
         queryset = queryset.select_related(u'inforequest')
         queryset = queryset.select_related(u'email')
         return queryset
 
+    def delete_constraints(self, objs):
+        constraints = []
+        for obj in objs:
+            if hasattr(obj.email, u'action'):
+                if obj.inforequest == obj.email.action.branch.inforequest:
```

Comment:
@martinmacko47 Dobre som porozumel podmienke, ktoru si napisal v issue?

Reply:
Podmienka vyzera spravne, ale neviem, ci je dobre ist na to po relacii smerom z emialu na action. Pretoze v buducnosti budeme mozno preratat obmedzenie, ze jeden email moze byt priradeny iba jednej akcii.

Lepsie by bolo tu kontrolu spravit opacne. Zobrat `obj.inforequest`, k nemu zobrat vsetky jeho akcie, k nim zobrat vsetky emaily k nim priradene. A potom pozriet, ci `obj.email` je medzi tymi emailami. Idealne ak sa da ten check tiez spravit tak, aby to bol iba jeden select.

Reply:
:ok: